server/src/users/users.controller.ts

```diff
@@ -35,10 +41,32 @@ export class UsersController {
     @Response({ passthrough: true }) response: ResponseT,
   ): Promise<void> {
     try {
-      // const user = await this.usersService.findByEmail(req.body.email);
-      // TODO send email the user
+      const user = await this.usersService.findByEmail(req.body.email);
+      if (!user) {
+        throw new Error();
```

Comment:
This should be
```suggestion
        return new NotFoundException(`foo bar`);
```

Reply:
left a comment inline for clarification
this is in a `try catch` so it doesn't really matter what error we throw here
and the catch responds 200 so that all those hackerz can't find out which emails are active or not on our service